pages/about.html

```diff
@@ -0,0 +1,72 @@
+    <script src="https://kit.fontawesome.com/3f00b2ab29.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header class="header-sect" >
+        
+        <div class="nav-bar">
+            <nav>
+                <button id="hide" class="hamburger" type="button"><i  id="bars" class="fa-solid fa-bars fa-2x icon-list"></i></button>
+                <ul class="mobile-menu" id="anchors">
+                    <li class="padding"><a class="pad" href="./index.html">Home</a></li>
+                    <li class="padding"><a class="pad" href="./about.html">About</a></li>
+                    <li class="padding"><a class="pad" href="#">Contact</a></li>
+                    <li class="padding"><a class="pad" href="#">Join</a></li>
+                  </ul>
+
+                <!-- desktop menu -->
+                <div class="desktop-menu">
+                    <img id="logo" src="../img/logo.jpg" alt="logo of company">
+                    <a href="./index.html">Home</a>
+                    <a href="./about.html">About</a>
+                    <a href="#">Program</a>
+                    <a href="#">Join</a>
+                    <a href="#">Sponsor</a>
+                    <a href="#">Campaign</a>
+
+                </div>
+                
+            </nav>
+        </div>
+            <div class="margin-header">
+                <h3 class="orange-color" id="header-text-1">
+                    "Hello people from all round the world"
+                </h3>
+                <h1 class="orange-color header-text2">GET INSPIRED BY THE BEST PEOPLE FROM AROUND THE WORLD ABOUT CLIMATE CHANGE</h1>
+            </div>
+            <p class="paragraph" id="paragraph-about">We are a team og people dedicated to fixing the global crisis on climate change. we hope weill join our cause to be the first generation to leave te planet better than we met it. A good heart is enough one person can make a differencce</p>
+            
+            <div class="contact">
+                <p>Please contact us per email for any further questions about  Climate change</p>
+                <h5><a href="#">lassanaa14@gmail.com</a></h5>
+
+            </div>
+    </header>
+    <section class="company-section">
+        <div class="spacing header">Climate 2022 </div>
+        <div class="bar"></div>
+        <div class="spacing desc">The logo of Amk was made in 2022 through the website Amklass logos</div>
+        <div > <img class="image" src="../img/logo.jpg" alt=""></div>
+    </section>
+    <section class="learn-section">
+        <h1 class="header">Learn more about climate change</h1>
+        <div class="bar"></div>
+        <p class="paragraph">Visit any of the links below if you want to leanr more about climate change</p>
+        <div class="pic-section">
+            <a id="red-pill" href="">Red pill</a>
+            <a id="blue-pill" href="">Blue pill</a>
+        </div>
+    </section>
+    <script src="../javascripts/scripts.js"></script>
+    
+</body>
```

Comment:
- [ ] Kindly be noticed that both pages contain the same navbar and footer.

Reply:
@ShahierNashaat  Sorry this isn't clear, Both nav bars are the same from my observations

Reply:
Hey @amklass11 I mean this footer 
![Screenshot from 2022-04-23 11-29-53](https://user-images.githubusercontent.com/33752735/164888859-06cf032d-a800-43e4-a396-5ce2c56bc7c4.png)

should also be in the about page.